go.mod

```diff
@@ -309,4 +309,7 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
-require github.com/google/gnostic-models v0.6.8 // indirect
+require (
```

Comment:
why this in a separate require block, even for indirect deps :thinking:

Reply:
`go mod tidy` or Goland inconsistency I think?

Reply:
Fixed